stack.yaml

```diff
@@ -19,6 +19,7 @@ extra-deps:
 - base58-bytestring-0.1.0
 - quickcheck-state-machine-0.6.0
 - command-0.1.1
+- QuickCheck-2.13.2
```

Comment:
As a rule of thumb I don't think we should pull non-LTS package versions unless they are really really necessary. Otherwise we will just cause pain for ourselves.

Reply:
@rvl wrote:

> As a rule of thumb I don't think we should pull non-LTS package versions unless they are really really necessary. Otherwise we will just cause pain for ourselves.

I've now removed this version.

Fortunately, the absence of a `NonPositive` QC modifier is easy to work around.